### PR TITLE
Support account creation on both VM & RPC

### PR DESCRIPTION
**Pull request by axic**
Don't merge yet.

Moved from https://github.com/chriseth/browser-solidity/pull/154

### CODE REVIEW

**Comment by chriseth** [comment]
Please rebase

**Comment by axic** [comment]
I think I'll split out the changes for using the "personal API" of web3.  I think that also requires changes to runTx. I'd postpone this PR until that is sorted.

**Comment by axic** [comment]
This is untested with web3 due to the personal account support - do not merge yet.

**Comment by axic** [comment]
Implementing this PR in a clean way depends on the outcome of https://github.com/ethereum/EIPs/issues/113

**Comment by chriseth** [comment]
ethereum/EIPs#113 was resolved

**Comment by axic** [comment]
@chriseth how should this work? Go through `personal` if it is available in the node (recent geth?), othwerise fall back to `eth` and disable account creation? Is it worth it? :)

**Comment by axic** [comment]
@chriseth also turns out we'll need to wait for web3.js 0.17.0 for the actual implementation

**Comment by axic** [comment]
This needs some testing, but I'm more confident that the latest version is suitable for merging soon.

It doesn't actually turn on `personal` support, it has to be explicitly required via the configuration option.

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "semistandard": "^7.0.0",
     "solc": "https://github.com/ethereum/solc-js",
     "tape": "^4.5.1",
-    "web3": "^0.16.0",
+    "web3": "https://github.com/ethereum/web3.js#35bcace7173fda742ddb1bba84c8a70f0fa849e0",
```
Comment (chriseth) [inline package.json]: Is that really needed? How will we keep track?

Comment (axic) [inline package.json]: That hash is the current version of `master`. Asked the web3.js team when a 0.17.0 release is coming, but no response yet.

**Comment by chriseth** [comment]
Please rebase

**Comment by axic** [comment]
I wonder why this wasn't merged in July. After the rework which happened since it is quite a big change to rebase to.

**Comment by axic** [comment]
@yann300 @chriseth can you test this and merge? In its current form it should have no adverse effect on anything.

**Comment by chriseth** [comment]
Tested and seems to have no effect on anything. Did not enable the switch. Did I see correctly that this does not have any UI components yet?

**Comment by axic** [comment]
@chriseth yes. There was no point complicating this PR any further. I think as a first step we could add a personal vs. non-personal mode switch to the UI.

**Pull request merged**